views/products/presentation/lightarti.tpl

```diff
@@ -0,0 +1,29 @@
+<h3>Anonymous Communication in Mobile Apps</h3>
+
+<img src="../../../resources/products/images/lightarti/logo.png" width="10%"
+     style="float: left; margin: 0.5em;" class="dark dark_pad"/>
+
+<p>Since the beginning of the commercial internet, and even more so with the start of mobile apps, 
+there is a tendency to collect as much data from the users as possible. This data is used to 
+monetize the users by selling it to other companies. But it can also be used by governments 
+to track users, even when the government said they will not do it.
+</p><p>
+On the network level, one of the best projects to guarantee the anonymity of the client is
+Tor. The Onion Router (Tor) relays the request of a client through a series of servers
+in such a way that no server has a full picture of the request. Unfortunately this advantage
+comes with a large setup cost for mobile applications: the client needs to download on a regular
+basis, every 3h, a full list of these servers.
```

Comment:
Where does the "every 3h" figure comes from? From what I could see, the consensus is updated every hour by the directory authorities:  https://metrics.torproject.org/glossary.html#consensus